puzzlejs/puzzle.js

```diff
@@ -1462,11 +1462,15 @@ function PuzzleGrid(puzzleEntry, options, container, puzzleId) {
 
         var changedGrids = [];
 
-        this.container.querySelectorAll(".inner-cell.extract").forEach(s => {
+        this.container.querySelectorAll(".inner-cell.extract, .inner-cell.link").forEach(s => {
             var extractId = s.getAttribute("data-extract-id");
+            var linkId = s.getAttribute("data-link-id");
+            var asel = []
+            if (extractId) asel.push("table:not(.copy-only) .extract[data-extract-id='" + extractId + "']");
+            if (linkId) asel.push("table:not(.copy-only) .cell[data-link-id='" + linkId + "']");
```

Comment:
this can be `.link`, not `.cell`, yes?

Reply:
Yup! Fixed. (Also updated changeWithoutUndo to use `.link` instead of `.cell`)